blockdag/reachability_test.go

```diff
@@ -624,26 +624,26 @@ func BenchmarkReindexInterval(b *testing.B) {
 		currentTreeNode := root
 		for i := 0; i < subTreeSize; i++ {
 			childTreeNode := newReachabilityTreeNode(&blockNode{})
-			_, err := currentTreeNode.addChild(childTreeNode)
+			_, err := currentTreeNode.addChild(childTreeNode, root)
 			if err != nil {
 				b.Fatalf("addChild: %s", err)
 			}
 
 			currentTreeNode = childTreeNode
 		}
 
-		remainingIntervalBefore := *root.remainingInterval
+		remainingIntervalBefore := *root.remainingIntervalAfter()
 		// After we added subTreeSize nodes, adding the next
 		// node should lead to a reindex from root.
 		fullReindexTriggeringNode := newReachabilityTreeNode(&blockNode{})
 		b.StartTimer()
-		_, err := currentTreeNode.addChild(fullReindexTriggeringNode)
+		_, err := currentTreeNode.addChild(fullReindexTriggeringNode, root)
 		b.StopTimer()
 		if err != nil {
 			b.Fatalf("addChild: %s", err)
 		}
 
-		if *root.remainingInterval == remainingIntervalBefore {
+		if *root.remainingIntervalAfter() == remainingIntervalBefore {
```

Comment:
Before and After here are confusing... (Before means before the `addChild `call while After `root.remainingIntervalAfter)` is related to interval logic